### PR TITLE
[IMP] MERPWH-603: Ventor Outgoing Routing: Fix spellings, module info and settings search

### PR DESCRIPTION
**Pull request by denisvalenchyts**

### CODE REVIEW

File: product_multiple_barcodes/__manifest__.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 {
     'name': 'Product Multiple Barcodes',
-    "version": "13.0.1.0.0",
```
Comment (okuryan) [inline product_multiple_barcodes/__manifest__.py:6]: If you are changing version of module, also update readme with version notes

File: product_multiple_barcodes/models/product_barcode_multi.py [hunk 2]
```diff
@@ -1,12 +1,24 @@
...
-    product_id = fields.Many2one('product.product', string='Product')
+    product_id = fields.Many2one(
+        'product.product', 
+        string='Product', 
+        required=True,
+        ondelete="cascade",
+    )
+
+    @api.constrains('name')
+    def _check_empty_name(self):
```
Comment (okuryan) [inline product_multiple_barcodes/models/product_barcode_multi.py]: Why need this and not just make name - required = True?

**Review by okuryan — CHANGES_REQUESTED** [review]

File: product_multiple_barcodes/__manifest__.py (hunk 1 above)
Comment (denisvalenchyts) [reply product_multiple_barcodes/__manifest__.py:6]: README.rst was added

File: product_multiple_barcodes/models/product_barcode_multi.py (hunk 2 above)
Comment (denisvalenchyts) [reply product_multiple_barcodes/models/product_barcode_multi.py]: Code was improved

File: product_multiple_barcodes/README.rst [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+***********************
+
+* Added field to define multiple additional barcodes of products
+* Added the search products by additional barcodes and internal reference
+
+13.0.1.1.0 (2020-04-30)
+***********************
+
+* Added mandatory typing the name of additional barcode
+* Added auto delete of additional barcodes when deleting products
```
Comment (ventor-dev) [inline product_multiple_barcodes/README.rst:13]: Last version should be at the top

Comment (denisvalenchyts) [reply product_multiple_barcodes/README.rst:13]: Done

**Pull request merged**